### PR TITLE
Allow installing the Hastexo XBlock from `master`/`main`

### PR DESCRIPTION
**Pull request by mrtmm**
For some cases, we might want to install the hastexo xblock from
`master` or `main`.
Tweak the conditionals so we don't break the playbook when doing so.

### CODE REVIEW

**Comment by fghaas** [comment]
Minor note: we might be renaming the default branch from `master` to `main` at some point in the future, so it probably doesn't hurt to test for the version being `master` *or* `main` already.

**Comment by mrtmm** [comment]
Thanks! I've included `main`.

File: playbooks/roles/hastexo_xblock/tasks/main.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     - openjdk-8-jdk-headless
     - tomcat8
     - maven
-  when: hastexo_xblock_version is version('5', '<')
+  when: hastexo_xblock_version not in ['master', 'main'] and hastexo_xblock_version is version('5', '<')
```
Comment (fghaas) [inline playbooks/roles/hastexo_xblock/tasks/main.yml:10]: You don't really need the `and` operator here. `when` conditions can include a list, which is automatically ANDed.

Thus:

```yaml
when:
  - hastexo_xblock_version not in ['master', 'main']
  - hastexo_xblock_version is version('5', '<')
```

Comment (mrtmm) [reply playbooks/roles/hastexo_xblock/tasks/main.yml]: Ack, done.

**Review by fghaas — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**